admin/client/src/legacy/LeftAndMain.js

```diff
@@ -370,15 +370,15 @@ $.entwine('ss', function($) {
      * @param object data - Any additional data passed through to `window.history.state`.
      * @param boolean forceReload - Forces the replacement of the current history state, even if the URL is the same, i.e. allows reloading.
      */
-    loadPanel: function (url, title = '', data = {}, forceReload, forceReferer = window.history.state.path) {
+    loadPanel: function (url, title = '', data = {}, forceReload, historyState = window.history.state) {
```

Comment:
The weird thing is that I can't see any changed `loadPanel()` call in this diff, so I've reverted https://github.com/open-sausages/silverstripe-framework/commit/72c92515af69f30631db05d4bd397fdcf71d474d